youtube_dl/extractor/porn91.py

```diff
@@ -36,28 +39,28 @@ def _real_extract(self, url):
             raise ExtractorError('91 Porn says: Daily limit 10 videos exceeded', expected=True)
 
         title = self._search_regex(
-            r'<div id="viewvideo-title">([^<]+)</div>', webpage, 'title')
+            r'<h4 class="login_register_header"[^>]+>([^<]+)</h4>', webpage, 'title')
```

Comment:
Here and at l.49, valid HTML could include whitespace or use quotes differently: see my comment at  https://github.com/ytdl-org/youtube-dl/pull/29884#discussion_r700962879.